content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md

```diff
@@ -0,0 +1,131 @@
+{{< /note >}}
+
+Por padrão, o `kubectl` procura por um arquivo de nome `config` no diretório `$HOME/.kube`
+
+Você pode especificar outros arquivos kubeconfig através da variável de ambiente `KUBECONFIG` ou adicionando a opção [`--kubeconfig`](/docs/reference/generated/kubectl/kubectl/).
+
+Para maiores detalhes na criação e especificação de um kubeconfig, veja o passo a passo [Configurar Acesso para Múltiplos Clusters](/docs/tasks/access-application-cluster/configure-access-multiple-clusters).
+
+
+<!-- body -->
+
+## Suportando múltiplos clusters, usuários e mecanismos de autenticação
+
+Imagine que você possua inúmeros clusters, e seus usuários e componentes se autenticam de várias formas. Por exemplo:
+
+- Um kubelet ativo pode se autenticar utilizando certificados
+- Um usuário pode se autenticar através de tokens
+- Administradores podem possuir conjuntos de certificados os quais provém acesso aos usuários de forma individual.
+
+Através de arquivos kubeconfig, você pode organizar os seus clusters, usuários, e namespaces. Você também pode definir contextos para uma fácil troca entre clusters e namespaces.
+
+
+## Contexto
+
+Um elemento de *contexto* em um kubeconfig é utilizado para agrupar parâmetros de acesso em um nome conveniente. Cada contexto possui três parâmetros: cluster, namespace, e usuário.
+
+Por padrão, a ferramenta de linha de comando `kubectl` utiliza os parâmetros do _contexto atual_ para se comunicar com o cluster.
+
+Para escolher o contexto atual:
+
+```shell
+kubectl config use-context
+```
+
+## A variável de ambiente KUBECONFIG 
+
+A variável de ambiente `KUBECONFIG` possui uma lista dos arquivos kubeconfig. Para Linux e Mac, esta lista é delimitada por vírgula. No Windows, a lista é delimitada por ponto e vírgula. A variável de ambiente `KUBECONFIG` não é um requisito obrigatório - caso ela não existir o `kubectl` utilizará o arquivo kubeconfig padrão localizado no caminho `$HOME/.kube/config`.
+
+Se a variável de ambiente `KUBECONFIG` existir, o `kubectl` utilizará uma configuração que é o resultado da combinação dos arquivos listados na variável de ambiente `KUBECONFIG`.
+
+## Combinando arquivos kubeconfig
+
+Para inspecionar a sua configuração atual, execute o seguinte comando:
+
+```shell
+kubectl config view
+```
+
+Como descrito anteriormente, a saída poderá ser resultado de um único arquivo kubeconfig, ou poderá ser o resultado da junção de vários arquivos kubeconfig. 
+
+Aqui estão as regras que `kubectl` utiliza quando realiza a combinação de arquivos kubeconfig:
+
+1. Se o argumento `--kubeconfig` está definido, apenas o arquivo especificado será utilizado. Apenas uma instância desta flag é permitida.
+
+   Caso contrário, se a variável de ambiente `KUBECONFIG` estiver definida, esta deverá ser utilizada como uma lista de arquivos a serem combinados.
+   Combine os arquivos listados na variável de ambiente `KUBECONFIG` de acordo com as seguintes regras:
```

Comment:
IMO here is a bit disconnected from the previous statement, maybe "[...] a serem combinados, seguindo o seguinte fluxo" and ignore the next statement :)